vmware-base/src/com/cloud/hypervisor/vmware/util/VmwareHelper.java

```diff
@@ -598,6 +601,19 @@ public static void setBasicVmConfig(VirtualMachineConfigSpec vmConfig, int cpuCo
         vmConfig.setGuestId(guestOsIdentifier);
     }
 
+    public static void addUSBController(VirtualMachineConfigSpec vmConfigSpec) {
```

Comment:
Yes, I'd second that idea.
let the method construct the spec of tyep VirtualDeviceConfigSpec for usb controller for us that could be added to vmConfig object at caller.